schematics/scully/src/ng-add/index.ts

```diff
@@ -134,17 +135,21 @@ const injectIdleService = (options: Schema) => (tree: Tree, context: SchematicCo
   }
 };
 
-const runScullySchematic = (options: Schema) => (tree: Tree, context: SchematicContext) => {
+const runBlogSchematic = (options: Schema) => (tree: Tree, context: SchematicContext) => {
```

Comment:
We decide remove the blog option, as a promp in the `ng add` can you remove the option? please

Reply:
Can we may merge this one and #177 first? As it will cause some conflicts otherwise I think

Reply:
In #177 I added some tests as well. So I would prefer to remove the option in #177 or to remove it in an extra PR / issue.